packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2889,20 +2889,34 @@ int _toLocationId(_Location location) {
   return id;
 }
 
-class _SerializationDelegate implements DiagnosticsSerializationDelegate {
-  _SerializationDelegate({
+/// A delegate that configures how a hierarchy of [DiagnosticsNode]s are
+/// serialized by the Flutter Inspector.
+@visibleForTesting
+class InspectorSerializationDelegate implements DiagnosticsSerializationDelegate {
+  /// Creates an [InspectorSerializationDelegate] that serialize [DiagnosticsNode]
+  /// for Flutter Inspector service.
+  InspectorSerializationDelegate({
     this.groupName,
     this.summaryTree = false,
     this.maxDescendentsTruncatableNode = -1,
     this.expandPropertyValues = true,
     this.subtreeDepth = 1,
     this.includeProperties = false,
     @required this.service,
+    this.addAdditionalPropertiesCallback,
   });
 
+  /// Service used by GUI tools to interact with the [WidgetInspector].
   final WidgetInspectorService service;
+
+  /// The `groupName` parameter is not needed but is specified to regularize the
```

Comment:
This doesn't really document what this property is for.

Reply:
Alright, I agree.